migrations/v8-MIGRATION.md

```diff
@@ -0,0 +1,48 @@
+# V8 API Migration Guide
+
+This latest release updates the SDK to use BillingClient 7. This version of BillingClient brings little change compared
+with BillingClient 5 which brought an entire new subscription model which resulted in large changes across the entire SDK. It did, however, increase the minSdk to 21 (Android 5.0) and removed the `ProrationMode` enum.
+
+Aside from that, we've also updated Kotlin to 1.8.22.
+
+The only modification at the API level involves removing `UpgradeInfo`, `ProrationMode` and related functions which was deprecated previously in V7 of our SDK but now has been removed completely since it depended on `ProrationMode` from the Billing Client which has been completely removed by Google.
```

Comment:
```suggestion
The only modification at the API level involves removing `UpgradeInfo` and `ProrationMode`. Related functions that were deprecated previously in V7 of our SDK have now been removed completely since they depended on classes not available anymore in billing client 7.0.0.
```